internal/app/machined/pkg/controllers/network/nftables_chain_test.go

```diff
@@ -351,12 +353,19 @@ func (s *NfTablesChainSuite) TestClampMSS() {
 
 	s.Require().NoError(s.State().Create(s.Ctx(), chain))
 
+	// several versions here for different version of `nft` CLI decoding the rules
 	s.checkNftOutput(`table inet talos-test {
 	chain test1 {
 		type filter hook input priority filter; policy accept;
 		meta nfproto ipv4 tcp flags syn / syn,rst tcp option maxseg size > 1380 tcp option maxseg size set 1380
 		meta nfproto ipv6 tcp flags syn / syn,rst tcp option maxseg size > 1360 tcp option maxseg size set 1360
 	}
+}`, `table inet talos-test {
+	chain test1 {
+		type filter hook input priority filter; policy accept;
+		meta nfproto ipv4 tcp flags & (syn | rst) == syn tcp option maxseg size > 1380 tcp option maxseg size set 1380
+		meta nfproto ipv6 tcp flags & (syn | rst) == syn tcp option maxseg size > 1360 tcp option maxseg size set 1360
+	}
```

Comment:
Is this unrelated and just a test fix?

Reply:
yes, see the comment, `nft` CLI of different versions decodes the rules a bit different way